src/lib/features/feature-toggle/tests/feature-toggle-last-seen-at.e2e.test.ts

```diff
@@ -113,3 +106,51 @@ test('response should include last seen at per environment for multiple environm
     expect(production.name).toBe('production');
     expect(production.lastSeenAt).toEqual('2023-10-01T12:34:56.000Z');
 });
+
+test('response should include last seen at per environment for multiple environments in /api/admin/archive/features', async () => {
+    const featureName = 'multiple-environment-last-seen-at-archived';
+    await setupLastSeenAtTest(featureName);
+
+    await app.request
+        .delete(`/api/admin/projects/default/features/${featureName}`)
+        .expect(202);
+
+    const { body } = await app.request.get(`/api/admin/archive/features`);
+
+    const featureEnvironments = body.features[0].environments;
+    const [def, development, production] = featureEnvironments;
+
+    expect(def.name).toBe('default');
```

Comment:
I'm wondering if we can make it more expressive using toMatchObject. Sometimes it's not possible but whenever I can write toMatchObject(listFieldsICareAbout) I prefer it for readability.